src/routes/index.jsx

```diff
@@ -33,6 +34,7 @@ const Routes = () => {
                         <Route path="/vacancies/:id" element={<ListCandidatesVacancies />} />
                         <Route path="/interviews" element={<Interviews />} />
                         <Route path="/interviews/create" element={<InterviewPlanning />} />
+                        <Route path="/postulants" element={<PostulationDetail />} />

```

Comment:
The route to match is `/postulants` but the header has `/postulations`. If you're going to change the header, remember you have to change the three files 😢 . Each file has different routes

Reply:
Yes the main route for postulations is in progress. This one is linked with the main one